webapp/src/components/App/Singlestudy/explore/Xpansion/Settings/SettingsForm.tsx

```diff
@@ -244,6 +257,7 @@ function SettingsForm(props: PropType) {
               handleChange("relaxed-optimality-gap", e.target.value)
             }
             sx={{ mb: 1 }}
+            InputProps={{ inputProps: { min: 0 } }}
```

Comment:
This wont work, you are trying to add a numeric value (number) constraint on a text (string) Field